setup.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+from setuptools import setup
+
+setup (
+    name = 'docker-sigprofiler', 
+    description = 'a implementation for sigprofiler docker CI', 
+    author = 'Jingwei Wang', 
+    url = 'www.example.com', 
+    author_email = 'jw32@sanger.ac.uk', 
```

Comment:
Shall we use cgphelp@sanger.ac.uk as the contact email? I'm sure you don't want to keep receiving questions even after you've moved to other projects.